matter-labs-tests/hardhat.config.ts

```diff
@@ -1,46 +1,74 @@
 import { HardhatUserConfig, subtask } from "hardhat/config";
+import { execSync } from "child_process";
 import "@nomicfoundation/hardhat-toolbox";
 import { TASK_COMPILE_SOLIDITY_GET_SOURCE_PATHS } from "hardhat/builtin-tasks/task-names";
+import '../../../hardhat-revive/packages/hardhat-resolc/src/index';
+import '../../../hardhat-revive/packages/hardhat-revive-node/src/index';
```

Comment:
```suggestion
import 'hardhat-resolc';
import 'hardhat-revive-node;
```
We should update these imports and also add the `hardhat-revive-node` plugin to `package.json`